manifests/params.pp

```diff
@@ -11,14 +11,12 @@
       $sumo_service_name  = 'collector'
       $syncsources    = '/etc/sumo.sources.d'
     }
-    'Debian': {
-      $sumo_package_name   = 'SumoCollector'
-      $sumo_service_config  = '/etc/sumo.conf'
-      $sumo_service_name  = 'collector'
-      $syncsources    = '/etc/sumo.sources.d'
-    }
-    'Ubuntu': {
-      $sumo_package_name   = 'sumocollector'
+    /^(Debian|Ubuntu)$/: {
```

Comment:
Ubuntu is not a valid `$osfamily`. The `$osfamily` for Ubuntu is `Debian`. 

The logic change here would work, though the case statement should be go back `'Debian': {`

Reply:
Fixed, that was a leftover.